scripts/scripts/questions.py

```
@@ -23,89 +23,27 @@
-        "word": "bok",
-        "words": ["studenten", "biblioteket"],
-        "blank1": "Studenten läser en ____ i biblioteket",
-        "blanks": [
-            "_____ läser en bok i biblioteket",
-            "Studenten läser en bok i _____",
-        ],
-        "example": """
-Example:
-Sentence: "Studenterna läser i biblioteket."
-Words: [{{"value":"Studenten","id":1}},{{"value":"läser","id":2}},{{"value":"en","id":3}},{{"value":"bok","id":4}},{{"value":"i","id":5}},{{"value":"biblioteket","id":6}}]
+PROMPT_TEMPLATE = """Your job is to select the most important words for language learners to focus on from the given sentence.
```

Pylint (3.3.7)

[convention] Line too long (126/100)

(C0301)